src/textbooks/data/thunk.js

```diff
@@ -59,3 +66,20 @@ export function editTextbookQuery(courseId, textbook) {
     }
   };
 }
+
+export function deleteTextbookQuery(courseId, textbookId) {
+  return async (dispatch) => {
+    dispatch(updateSavingStatus({ status: RequestStatus.IN_PROGRESS }));
+    dispatch(showProcessingNotification(NOTIFICATION_MESSAGES.deleting));
+
+    try {
+      await deleteTextbook(courseId, textbookId);
+      dispatch(deleteTextbookSuccess(textbookId));
+      dispatch(updateSavingStatus({ status: RequestStatus.SUCCESSFUL }));
+      dispatch(hideProcessingNotification());
```

Comment:
Move `dispatch(hideProcessingNotification());` to `finally` since it in both `try` and `catch`

Reply:
Fixed